tests/smoke_test.go

```diff
@@ -176,6 +179,85 @@ func TestLanguageGenerateSmoke(t *testing.T) {
 	}
 }
 
+func TestPackageGetSchema(t *testing.T) {
+	t.Parallel()
```

Comment:
Shouldn't be parallel, you need to mutate PULUMI_HOME

Reply:
Done ✅